lib/symboldatabase.cpp

```diff
@@ -2629,6 +2629,37 @@ bool Function::argsMatch(const Scope *scope, const Token *first, const Token *se
     return false;
 }
 
+static bool isUnknownType(const Token* start, const Token* end)
+{
+    while (Token::Match(start, "const|volatile"))
+        start = start->next();
+    if (Token::Match(start, ":: %name%"))
```

Comment:
Could one use a utility function like `skipScopeIdentifiers(const Token* tok)` here?

Reply:
I dont know, I didnt write this code, I am just moving it to a separate function.

Reply:
thanks @amai2012 !! I will test that..